resources/local/chainspec.toml

```diff
@@ -53,6 +53,10 @@ finality_threshold_percent = 10
 # Integer between 0 and 255. The power of two that is the number of milliseconds in the minimum round length, and
 # therefore the minimum delay between a block and its child. E.g. 14 means 2^14 milliseconds, i.e. about 16 seconds.
 minimum_round_exponent = 12
+# Integer between 0 and 255. Must be greater than `minimum_round_exponent`. The power of two that is the number of
```

Comment:
Maybe "greater or equal"?
And "the power of two of that"?

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/88f1c9e89dd61c6f169a305d466f0b84e439d147